Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
-import './Card.css';
+
+const CardBanner = () => (
+  <img
+    className="img-background"
```

Comment:
All you need to do to constrain the image is to use this class in your CSS and add `width: 100%`.